From 7bcb69812d7790b74bbf35e0bd5d48396ce01271 Mon Sep 17 00:00:00 2001
From: Praveen Diwakar <praveen.diwakar@intel.com>
Date: Thu, 29 May 2014 20:31:43 +0530
Subject: [PATCH 03/24] ASoC: Intel: Debug: Update sram dump offset

Mailbox offset updated for ia->lpe and lpe->ia dump.

Issue: GMINL-3680

Change-Id: I3a05cc306258645b7a39819a75e13771a777d5ac
Signed-off-by: Praveen Diwakar <praveen.diwakar@intel.com>
---
 sound/soc/intel/sst/sst_debug.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sound/soc/intel/sst/sst_debug.c b/sound/soc/intel/sst/sst_debug.c
index 837da70..919faee 100644
--- a/sound/soc/intel/sst/sst_debug.c
+++ b/sound/soc/intel/sst/sst_debug.c
@@ -311,8 +311,8 @@ static ssize_t sst_debug_sram_ia_lpe_mbox_read(struct file *file,
 	if (ret)
 		return ret;
 	ret = copy_sram_to_user_buffer(user_buf, count, ppos, IA_LPE_MAILBOX_DUMP_SZ,
-				       (u32 *)(drv->mailbox + SST_MAILBOX_SEND),
-				       SST_MAILBOX_SEND);
+			       (u32 *)(drv->ipc_mailbox + SST_MAILBOX_SEND),
+			       SST_MAILBOX_SEND);
 	sst_pm_runtime_put(drv);
 	return ret;
 }
@@ -335,8 +335,8 @@ static ssize_t sst_debug_sram_lpe_ia_mbox_read(struct file *file,
 		return ret;
 
 	ret = copy_sram_to_user_buffer(user_buf, count, ppos, LPE_IA_MAILBOX_DUMP_SZ,
-				       (u32 *)(drv->mailbox + drv->mailbox_recv_offset),
-				       drv->mailbox_recv_offset);
+		       (u32 *)(drv->ipc_mailbox + drv->mailbox_recv_offset),
+		       drv->mailbox_recv_offset);
 	sst_pm_runtime_put(drv);
 	return ret;
 }
-- 
1.9.1

